src/ClientData/include/ClientData/CaptureData.h

```diff
@@ -111,9 +111,9 @@ class CaptureData {
   }
 
   [[nodiscard]] const std::string& GetThreadName(uint32_t thread_id) const {
-    static const std::string kEmptyString;
+    static const std::string empty_string;
```

Comment:
The style guide says: "Variables declared constexpr or const, and whose value is fixed for the duration of the program, are named with a leading "k" followed by mixed case."

Reply:
You were right. I thought it would be a limitation of clang-tidy, but that was actually false. I added the rule and applied it. We now have "static const" and "constexpr" as kCamelCase, all other variables should be snake_case

Reply:
So this can be changed back?

Reply:
This is so much better now by the way.